src/test/yaml.completion.test.ts

```diff
@@ -0,0 +1,161 @@
+'use strict';
+
+import * as vscode from 'vscode';
+import * as chai from 'chai';
+import { getDocUri, activate } from './helper';
+
+const expect = chai.expect;
+
+describe('Should do completion in Camel URi after "ti" },', () => {
+	const yamlSchemaUri: vscode.Uri = getDocUri('camelk-yaml-schema.json');
```

Comment:
maybe worth using a name specifying that it is "temporary/intermediate/in progress" schema.

Reply:
I don't see any benefit of changing the name, tbh.

Reply:
it allows to know that this schema can be updated when a new version is available.

Reply:
latest version of code references a URL for the schema. its no longer a local file in the repo. just wait for the next commit.